### ISSUE TITLE
Ecoregion: factors need hyperlink possibilities

### ISSUE THREAD

**Issue by ldford** [Ecotrust/ocs-wp#25]
There are several items we need to be able to hyperlink in the Ecoregion "factors" section for both Factor Descriptions and Approaches. See highlighted text

![image](https://cloud.githubusercontent.com/assets/6453589/10180946/2e913f14-66c7-11e5-8edd-51539280dcfa.png)

**Comment by willthemoor** [comment]
**tl;dr**: This can't be fixed in time for alpha due to a bug within an included library.

The problem here is the [bug in CMB2](https://github.com/WebDevStudios/CMB2/issues/462) (the library we're using to generate the fields in the admin) that does not allow a normal WYSIWYG box to be inserted into a repeater field (repeater = any field with an "add another"). As of today, that bug still exists. 

There is work being done on it but it's doubtful it will be published soon. https://github.com/WebDevStudios/CMB2/pull/431 and https://github.com/WebDevStudios/CMB2/issues/264 (has a link towards the end to a WIP branch with it *mostly* working)

For now, there's only two ways to insert links: 
1) `<a href="manually">by typing html</a>`  
2) use the main content area to insert a link with the toolbar. Then go to the 'text' view and copy the generated link. Go to the factor description and paste it in.
